### DISCUSSION TITLE
My layout is duplicated

### DISCUSSION THREAD

**Discussion by CharlesOnAir** [nuxt/nuxt#16698] [Questions]
Hello,
I have a rather particular situation, my layout works correctly when I access my pages.

However, when a user wishes to access a page that has not been authorized, I redirect the user to another page using the "navigateTo" function.

When I'm on the page, after being redirected by "navigateTo", the layout of my previous page is added to my current page. (see screenshot).

Do you have any idea where to look for fix this problem ?

Thanks for your help.

![193322566-6030fdc0-9c87-45ba-8877-43bc62f96105](https://user-images.githubusercontent.com/16211129/193402892-ec9ce9c0-fda2-4a75-967b-9a0a3ef9a6b7.png)
![193322722-18bafb7d-b59e-4ca1-808b-b58e38199897](https://user-images.githubusercontent.com/16211129/193402897-9fb68aa3-27e4-4503-835c-7132def97836.png)

The page where I am redirecting :
<img width="861" alt="193322730-d31a8c1f-ba04-450f-ab63-8871f147a6a4" src="https://user-images.githubusercontent.com/16211129/193402903-db34257c-72db-4cf7-b195-134a52d39ac0.png">

The page that contains the double layout :
<img width="930" alt="193322766-84834be9-3851-4542-b69b-14746fc2fde0" src="https://user-images.githubusercontent.com/16211129/193402908-45b1f7bf-560f-4529-82c2-eb52964b1cd0.png">

**Comment by TheAlexLichter** [comment]
Could you provide a small reproduction repository? That could help debugging 😋 

Also: Any errors in the console?

**Comment by jiblett1000** [comment]
Same here. Might be getting fixed by this PR. Not sure if/when that might happen :/ https://github.com/nuxt/framework/pull/7940#issuecomment-1264939541
